<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>语音输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>转换为</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏输出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>转换为</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>播放(F10)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_left">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>免费使用次数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_left_times">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="btn_chooseMusic2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>选择音乐</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前背景音乐：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMusicPath2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>491</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/documents/test.mp3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>音量（0-1）</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_music_volumn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_setMusic">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>370</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>设置音乐</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMusicVolum">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前：1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>发泄对象</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_target">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>声音来源</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_voice">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板一</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板二</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板三</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板四</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板五</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_template6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>模板六</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_setTemplate">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>设置模板</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_content">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>751</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前背景音乐：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMusicPath">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>491</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/documents/test.mp3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_circulate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>循环播放</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_background">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>背景音乐</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_saveRecord">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>储存录音</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_chooseMusic">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>选择背景音乐</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_defaultMusic">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>使用默认背景音乐</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>340</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>采集(F11)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_voice_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>声音来源</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>语速（0-15整数）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>语调（0-15整数）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>音量（0-15整数）</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_set_voice_speed">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_set_voice_tune">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_set_voice_volume">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="settingChange">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>320</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingDefault">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>320</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>恢复默认</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSpeed">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTune">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVolume">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page5">
     <widget class="QPushButton" name="btn_vip">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>340</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>激活</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_activate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>输入激活码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_activate_status">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>未激活</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>无</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>售后qq群：626913648</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>备注请输入需要激活的账号，忘记输入可加qq群找群主</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>321</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/res/pay2.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/res/paywx.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page6">
     <widget class="QPushButton" name="btn_b2g">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>男变女</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_g2b">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>女变男</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_confirm_change_voice">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>当前：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cv">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>默认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cv_default">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>默认</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>音调：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cv_le">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_pitch">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="topBtn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>放歌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>喷人</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>AI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>激活</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>变声</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
